mars/services/task/execution/ray/executor.py

```diff
@@ -73,6 +73,21 @@ def _optimize_subtask_graph(subtask_graph):
     return _optimize_physical(subtask_graph)
 
 
+async def _cancel_ray_task(obj_ref, kill_timeout: int = 3):
+    ray.cancel(obj_ref, force=False)
+    try:
+        await asyncio.to_thread(ray.get, obj_ref, timeout=kill_timeout)
+    except ray.exceptions.TaskCancelledError:  # pragma: no cover
+        logger.info("Cancel ray task %s successfully.", obj_ref)
+    except BaseException as e:
+        logger.info(
+            "Failed to cancel ray task %s with exception %s, force cancel the task by killing the worker.",
```

Comment:
This line is too long, can you split it into two lines?

Reply:
Ok